.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "deps/onig"]
+	path = deps/onig
```

Comment:
This is a bit of a nitpick, but would you mind using 2-space indentation here instead of tabs? :bow: